package.json

```diff
@@ -9,9 +9,9 @@
         "url": "https://github.com/KartikTalwar/gmail.js.git"
     },
     "scripts": {
-        "lint": "./node_modules/.bin/eslint src/*.js",
-        "tsc": "./node_modules/.bin/tsc",
-        "test": "./node_modules/.bin/mocha test/test.*.js",
+        "lint": "npx eslint src/*.js",
+        "tsc": "npx tsc",
+        "test": "npx mocha test/test.*.js",
```

Comment:
Are these changes strictly speaking required for this PR?

Reply:
The scripts, as they were, didn't work in my case, so I've replaced them with `npx`, which first looks for local versions of the respective modules. I can revert them if it's not ok for some reason.

Reply:
If this works better, I’m for that. 

That said, this would make npx a new devDependency we didn’t have before, and it will have to be declared as such too.

Reply:
`npx` comes with `npm`, https://docs.npmjs.com/cli/v7/commands/npx. So no new dependency.

Reply:
TIL. No problem then!